lib/config.ps1

```diff
@@ -21,13 +21,14 @@
     "computer"
     # "custom_time"  # use custom info line
     "uptime"
-    "pkgs"
+    "resolution"
+    # "pkgs" # might take slightly more time
```

Comment:
I don't think this should be removed unless it's significantly slow which it shouldn't as its really just listing directories

Reply:
I tried to measure how long each section takes. Scoop report was the slowest for me (as it's doing a directory traversal). As I said, I was under the assumption that the default config should include (relatively) faster operations, and resolution is just a variable that needs to be printed.

Reply:
Actually, the reason `pkgs` is slow is because of `Get-Command` which takes 160ms if it fails (e.g. if you don't have chocolatey installed) and `scoop which scoop` which takes about 170ms. Both of which can be partially eliminated, which is something I'll address soon.

Reply:
Sure, that'll be great!